opsdroid/connector/github/tests/test_connector_github.py

```diff
@@ -97,76 +88,69 @@ async def test_listen(connector):
     assert await connector.listen() is None
 
 
+@pytest.mark.add_response(
+    "/repos/opsdroid/opsdroid/issues/1/comments", "POST", None, status=201
```

Comment:
The downside of this approach is having to define this explicitly outside the test, before it was defined from variables used in the tests...

Reply:
At least in this file, with the strings reused in so many tests, I think it was cleaner to pull them out into some constants and build them all once.